Dockerfile-development

```diff
@@ -2,16 +2,10 @@ FROM node:8.14.0
 
 ENV HOME "/home/xpub"
 
-RUN mkdir -p ${HOME}
+VOLUME ["/home/xpub"]
```

Comment:
Is this necessary ? we are specifying the volume in the docker-compose.override.yml file

Reply:
actually after thinking about it is it a way of declaring that the container should have volume mounted?

Reply:
It probably isn't - but I like being explicit ... because now this container will not work (as I removed the copy commands) without a mounted volume.